build/Makefile

```
@@ -312,14 +313,14 @@ install: $(ensure-build-image) install-custom-pull-secret
 		--set agones.controller.logLevel=$(LOG_LEVEL) \
 		--set agones.crds.cleanupOnDelete=$(CRD_CLEANUP) \
 		--set agones.featureGates=$(FEATURE_GATES) \
+		--set agones.allocator.http.loadBalancerIP=$(EXTERNAL_IP) \
 		$(HELM_ARGS) \
 		agones $(mount_path)/install/helm/agones/
 
-update-allocation-certs: EXTERNAL_IP ?= $(shell $(DOCKER_RUN) kubectl get services agones-allocator -n agones-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 update-allocation-certs: NAMESPACE ?= default
 update-allocation-certs:
```

```suggestion
update-allocation-certs: install
```

Instead of having it inline below.

---

Thanks for the suggestion. Applied the change.